mkdocs.yml

```diff
@@ -138,6 +138,8 @@ nav:
           - devices/rigetti-qcs.md
   - Qubit Routing:
       - qubit-routing/qaoa-qubit-routing.md
+  - Error Mitigation:
+    - error-mitigation/spam-twirling.md
```

Comment:
Did you try to build the website? The indent seems off here!

Reply:
yes, it works fine for me

Reply:
![Screenshot 2023-04-28 at 15 51 52](https://user-images.githubusercontent.com/22844056/235088171-738888d6-9e17-4f79-9334-eae13d0039cd.png)

Reply:
yes, it seems to be running fine but I also fixed the yml file